### ISSUE TITLE
Does not report the completion correctly

### ISSUE THREAD

**Issue by mudrd8mz** [LafColITS/moodle-local_feedbackviewer#12]
I have two Feedback in a course. As a student, I completed them both. When I visit each of them as a student, I can see

![image](https://user-images.githubusercontent.com/85831/30203245-c46892b4-9481-11e7-996a-c4a921af495c.png)

But when I visit the plugin, I see something different:

![image](https://user-images.githubusercontent.com/85831/30203266-d76b41fe-9481-11e7-94df-c79aa24dbd05.png)

I see that as a student and also as a teacher.

Are there some special requirements on Feedbacks or something?

**Comment by mudrd8mz** [comment]
Just in case it matters, both feedbacks consist of a single multichoice question (rated in one of them).

**Comment by mudrd8mz** [comment]
Oh I see now. In my case, the feedbacks had "Record user names" set to "Anonymous". So the field "anonymous_response" in the table "mdl_feedback_completed" is set to 1 and the `build_report()` method then displays the "not_completed_yet" string.

I believe this should be fixed and improved.

# For the student, their own responses should be always displayed. It is "My" replies anyway, the anonymity does not apply here.
# For the teacher, the report could first check the feedback setting. If the feedback is anonymous, it should communicate that clearly (explaining that data for anonymous feedbacks can't be displayed) instead of reporting that it was not completed yet.

**Comment by mackensen** [comment]
Yes, that use case really isn't supported, but it should be made explicit.
